### ISSUE TITLE
DX12, 11, and 10 Can Randomly Get Disabled

### ISSUE THREAD

**Issue by JohnnyTex** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#305]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Application is not cracked, modded and use the latest patch

### Application [Required]

For all games

### Processor / Processor Number [Required]

i5 12600k

### Graphic Card [Required]

ARC A770 LE

### GPU Driver Version [Required]

- [X] 31.0.101.4335
- [ ] 31.0.101.4314
- [ ] 31.0.101.4311
- [ ] 31.0.101.4257
- [ ] 31.0.101.4255
- [ ] 31.0.101.4148
- [ ] 31.0.101.4146
- [ ] 31.0.101.4125
- [ ] 31.0.101.4123
- [ ] 31.0.101.4091
- [ ] 31.0.101.4090
- [ ] 31.0.101.4034
- [ ] 31.0.101.4032
- [ ] 31.0.101.3975

### Rendering API [Required]

- [ ] Vulkan
- [ ] OpenGL
- [X] DirectX12
- [X] DirectX11
- [X] DirectX10
- [ ] DirectX9
- [ ] Not applicable

### Windows Build Number [Required]

- [X] Windows 11 22H2
- [ ] Windows 11 21H2
- [ ] Windows 10 22H2
- [ ] Windows 10 21H2
- [ ] Other (Please specify)

### Intel System Support Utility report

[postdxcrash.txt](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/11369774/postdxcrash.txt)

### Description and steps to reproduce [Required]

Described in greater detail on Horizon and Discord through the following links:
Horizon - https://inteltesting.centercode.com/project/feedback/view.html?cap=09589d2c-98db-48ae-80c5-811c73851a30&uf=b9b6a044-4e27-4791-8a3b-7daba71f5262&slsid=7771d53a-a1ef-465d-8d9c-7ba938bcf909 
Discord - https://discord.com/channels/554824368740630529/1076839214181781524 

Basically, either before, during, or after a game is played, all DX levels 10 to 12 will randomly get disabled. Sometimes it's presaged by sudden and drastic performance loss. The game that seems to happen the most to me in is Cold War.

**Comment by Karen-Intel** [comment]
Hey @JohnnyTex we already have checked internally and the Horizon team is following this case quite close (with a bug already opened), so please post any related info in the Horizon thread
@IGCIT can you help us close this post as dupe? It is being followed already in another debug platform
https://inteltesting.centercode.com/project/feedback/view.html?cap=09589d2c-98db-48ae-80c5-811c73851a30&uf=b9b6a044-4e27-4791-8a3b-7daba71f5262&slsid=7771d53a-a1ef-465d-8d9c-7ba938bcf909

Thanks

Karen

**Comment by Arturo-Intel** [comment]
A quick update:
The engineer team is well aware of this problem, so its quite possible that we have a fix soon, remember it takes 3 -6 months to release the fix to public, you may found more details about this in the horizon platform case.
